sherpa/astro/utils/src/_region.cc

```diff
@@ -379,6 +379,18 @@ PyInit__region(void)
     Py_DECREF(&pyRegion_Type);
     Py_DECREF(m);
   }
+
+  // Add a symbol to indicate whether the module was built with the
+  // USE_CXCDM_PARSER macro set.
+  //
+  PyModule_AddIntConstant(m, "USE_CXCDM_PARSER",
```

Comment:
The #ifdef preprocessor wraps lexical value C commands within the conditional text segment. As such, it should be written as:

#ifdef USE_CXCDM_PARSER
	PyModule_AddIntConstant(m, "USE_CXCDM_PARSER", 1);
#else
	PyModule_AddIntConstant(m, "USE_CXCDM_PARSER", 0);
#endif

Alternatively, have you tried using?

    PyModule_AddIntMacro(m, USE_CXCDM_PARSER);

Reply:
So, I'm not sure what `PyModule_AddIntMacro` does when the macro isn't defined (or is defined but has "no value").

I guess we could change how the code is built so that rather than sending in `-DUSE_CXCDM_PARSER` only when we want the macro set we'd send in either `-DUSE_CXCDM_PARSER=0` or `-DUSE_CXCDM_PARSER=1` but that's a larger change that I was hoping to avoid.

Unless there's a reason I would prefer to go with the initial suggestion - i.e.

```
#ifdef USE_CXCDM_PARSER
PyModule_AddIntConstant(m, "USE_CXCDM_PARSER", 1);
#else
PyModule_AddIntConstant(m, "USE_CXCDM_PARSER", 0);
#endif
```

Reply:
I can confim that `PyModule_AddIntMacro` needs the macro to exist, so the easiest change is to duplicate the lines.